tests81/Unit/Models/Enums/StatusEnum.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace Backpack\CRUD\Tests81\Unit\Models\Enums;
+
+    enum StatusEnum: string
+    {
+        case DRAFT = 'DRAFT';
+        case PUBLISHED = 'PUBLISHED';
```

Comment:
I think it's better for the "name" and the "value" to be different so troubleshooting will be easier later.

Reply:
Thanks, I did it! 👍